ui/src/app/domains/views/DomainsList/DomainsTable/DomainsTable.tsx

```diff
@@ -7,19 +7,41 @@ const DomainsTable = (): JSX.Element => {
   const domains = useSelector(domainSelectors.all);
 
   const headers = [
-    { content: "Domain" },
-    { content: "Authoriatative" },
-    { content: "Hosts", className: "u-align--right" },
-    { content: "Total records", className: "u-align--right" },
-    { content: "Actions", className: "u-align--right" },
+    {
+      content: "Domain",
+      sortKey: "name",
+      "data-test": "domain-name-header",
+    },
+    {
+      content: "Authoriatative",
+      sortKey: "authoritative",
+    },
+    {
+      content: "Hosts",
+      sortKey: "hosts",
+      className: "u-align--right",
+    },
+    {
+      content: "Total records",
+      sortKey: "records",
+      className: "u-align--right",
+    },
+    {
+      content: "Actions",
+      className: "u-align--right",
```

Comment:
Was it intentional to not add a `sortKey` here?

Reply:
That's the column for the actions menus, so there's nothing to sort.

Reply:
Oh right, of course 🤦